src/kOS/AddOns/RemoteTech/RemoteTechHook.cs

```diff
@@ -24,6 +24,15 @@ public static IRemoteTechAPIv1 Instance
             }
         }
 
+        /// <summary>
+        /// True if Not ONLY does the vessel exist and Remote Tech is enabled,
+        /// but ALSO, the vessel is *loaded* and has a ModuleSPU.  Note that if
+        /// the vessel does have a ModuleSPU but is *NOT* loaded (i.e. it's outside
+        /// the physics bubble), then this will return False *no matter what*,
+        /// because Remote Tech removes the Flight Computer from all distant vessels.
+        /// </summary>
+        /// <param name="vesselId"></param>
+        /// <returns></returns>
```

Comment:
No change here - just felt it was important to mention this rather important fact for any future mantainers.